server.js

```diff
@@ -28,7 +28,7 @@ const viewsPath = path.resolve(rootPath, 'src/server/views');
 const isProduction = process.env.NODE_ENV === 'production';
 const { applicationPort, sqs, oauth, publicKey } = appConfig;
 // AWS SQS Configuration
-aws.config.update({ region: sqs.region });
+aws.config.update({ region: process.env.SQS_REGION });
```

Comment:
Honestly, I kinda prefer it using `sqs.region` — even when using the `.env` file. BUT, I really don't mind — it's not a big deal.